README.md

```diff
@@ -287,3 +287,35 @@ Follow [these instructions](https://www.digitalocean.com/community/tutorials/how
 1. From a shell with kubectl access to the cluster, cd into `BraveSensor-Server` repository and run `kubectl apply -f manifests/redis_storage.yaml`
 2. Install redis from the bitnami/redis chart by running `helm install <environment name>-sensor-redis bitnami/redis --namespace redis --set persistence.storageClass=redis-storage --set usePassword=false --set cluster.enabled=false`
 3. Run `kubectl get svc --namespace=redis` to get the IP of your new redis instance to use as an environment variable
+
+# Particle API access
+
+The Brave Siren functions use the Particle API to trigger the Siren when a `DURATION` or `STILLNESS` event occurs. This requires the access token (`PARTICLE_ACCESS_TOKEN`) for an API user with permission to call functions in the product group (`PARTICLE_PRODUCT_GROUP`).
+
+To generate this token:
```

Comment:
is there a reason to do this other than setting up a new environment?

Reply:
Setting up a new environment or if we ever want to rotate the token for better security. In fact, I've written these instructions as though there already is a token in prod that we don't want to break, so it's more for the rotation case.